bot/exts/fun/duck_game.py

```diff
@@ -245,33 +248,41 @@ async def on_message(self, msg: discord.Message) -> None:
         if answer in game.solutions:
             game.claimed_answers[answer] = msg.author
             game.scores[msg.author] += CORRECT_SOLN
-            await self.display_claimed_answer(game, msg.author, answer)
+            await self.append_to_found_embed(game, f"{str(answer):12s}  -  {msg.author.display_name}")
```

Comment:
Isn't `answer` a tuple? Maybe we could format it more prettier?

Reply:
Do you have a suggestion? It is a tuple, but the answers are given in the form of tuples anyway. I'm not sure what else you would display it as.

Reply:
Well tuples become `(item, item, item, item)` but we could format it ourselves. Maybe as `item, item, item, item and item`.

Reply:
I think the information is more clear with the parentheses.

Reply:
I agree, parentheses make sense here.